2020-05-06 14:08:02 [scrapy] ERROR: Error while obtaining start requests
Traceback (most recent call last):
  File "/usr/lib/python3.8/site-packages/scrapy/core/engine.py", line 127, in _next_request
    request = next(slot.start_requests)
  File "/home/lalit/.local/lib/python3.8/site-packages/scrapy_splash/middleware.py", line 167, in process_start_requests
    for req in start_requests:
  File "/run/media/lalit/Work/scrapy-googlescholar/topl_project/topl_project/spiders/1.py", line 52, in start_requests
    url=self.url
AttributeError: 'CitSpider' object has no attribute 'url'
2020-05-06 14:08:05 [scrapy] DEBUG: Crawled (200) <GET https://scholar.google.com/scholar?oi=bibs&hl=en&cites=13676029927784182995> (referer: None)
2020-05-06 14:08:05 [scrapy] INFO: Closing spider (finished)
2020-05-06 14:08:05 [scrapy] INFO: Dumping Scrapy stats:
{'downloader/request_bytes': 351,
 'downloader/request_count': 1,
 'downloader/request_method_count/GET': 1,
 'downloader/response_bytes': 31432,
 'downloader/response_count': 1,
 'downloader/response_status_count/200': 1,
 'elapsed_time_seconds': 2.739293,
 'finish_reason': 'finished',
 'finish_time': datetime.datetime(2020, 5, 6, 8, 38, 5, 659518),
 'log_count/DEBUG': 1,
 'log_count/ERROR': 1,
 'log_count/INFO': 7,
 'memusage/max': 64352256,
 'memusage/startup': 64352256,
 'response_received_count': 1,
 'scheduler/dequeued': 1,
 'scheduler/dequeued/memory': 1,
 'scheduler/enqueued': 1,
 'scheduler/enqueued/memory': 1,
 'start_time': datetime.datetime(2020, 5, 6, 8, 38, 2, 920225)}
2020-05-06 14:08:05 [scrapy] INFO: Spider closed (finished)
